.make/license/updateLicenses.php

```diff
@@ -19,17 +19,25 @@
  * The target license file path is in `$xmlFilePath`
  */
 
-$iTopFolder = __DIR__ . "/../../" ;
-$xmlFilePath = $iTopFolder . "setup/licenses/community-licenses.xml";
+$iTopFolder = __DIR__."/../../";
+$xmlFilePath = $iTopFolder."setup/licenses/community-licenses.xml";
+
+$jqExec = shell_exec("jq --help"); // --help param is mandatory otherwise the script will freeze
+if ((null === $jqExec) || (false === $jqExec)) {
+	echo "/!\ JQ is required but cannot be launched :( \n";
+	echo "Check this script PHPDoc block for instructions\n";
+	die - 1;
```

Comment:
```suggestion
	die - 1;
```

Like for `exit()`, return code should be in parenthesis. I don't recall the symptom if we don't but we had similar issues with CLI scripts within the SaaS team if I recall correctly. Poke @odain-cbd

Reply:
I also don't like this notation..

Reply:
Woops, thanks ! 
Fixed in 6b52da4d